### PR TITLE
testing/py-geoip-legacy && geoip-database: new aports

### PR DESCRIPTION
**Pull request by itoffshore**
* legacy `py-geoip` currently needed by `nxapi`

* legacy `geoip-database` is still [packaged by Debian](https://packages.debian.org/stretch/geoip-database)

### CODE REVIEW

File: testing/py-geoip-legacy/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=py-geoip-legacy
```
Comment (jirutka) [inline testing/py-geoip-legacy/APKBUILD]: Why have you named it like this when the upstream name is geoip-api-python (so py-geoip-api)?

File: testing/py-geoip-legacy/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+pkgrel=0
+pkgdesc="MaxMind GeoIP Legacy Database"
+url="https://pypi.python.org/pypi/GeoIP"
+arch="all"
+license="GPL2"
+depends=""
+makedepends="python2-dev python3-dev py-setuptools geoip-dev"
+checkdepends="py-nose"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
+source="$pkgname-$pkgver.tar.gz::https://github.com/maxmind/geoip-api-python/archive/v1.3.2.tar.gz"
```
Comment (jirutka) [inline testing/py-geoip-legacy/APKBUILD]: Version number must be parametrized.

File: testing/geoip-database/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+pkgname=geoip-database
+pkgver=20170714
+pkgrel=0
+pkgdesc="GeoIP library (legacy country databases)"
+url="http://maxmind.com"
+arch="noarch"
+license="CC-4.0"
+depends=""
+options="!check"
+source="http://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz
```
Comment (itoffshore) [inline testing/geoip-database/APKBUILD]: not much can be done about that - & is why the Debian version number is also a date

**Comment by itoffshore** [comment]
resubmitted as https://github.com/alpinelinux/aports/pull/1925

**Pull request closed without merging**